app/assets/stylesheets/welcome.scss

```diff
@@ -31,11 +31,23 @@ $backdrop : #C9D6AE;
   font-size: 80px;
   color: white;
   margin-left: 10px;
-  word-wrap: break-word;
+  width: 560px;
 }
 
 #welcome-description {
   font-size: 24px;
   color: white;
   margin-left: 10px;
+  width: 600px;
+}
+
+#app-info {
+  font-size: 24px;
+  color: black;
+  margin-top: 20px;
+  margin-left: 10px;
+  background: rgba(204, 204, 204, 0.7);
```

Comment:
Color literals like `rgba(204, 204, 204, 0.7)` should only be used in variable declarations; they should be referred to via variable everywhere else.